src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -21,7 +22,7 @@ const generateInitialOptions = (meta) => {
               - data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index - 1].y)
               .toFixed(2);
             }
-            return `Delta: ${delta}`;
+            return `${data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index].y} (${higherOrLower}) \n Delta: ${delta}`;
```

Comment:
Generally if you want a line break, you need to use the `<br />` tag, although in this situation I think it'd be better to have Delta as its own string on a separate line and then wrap both strings in one `<span>` or `<div>`.

Reply:
@sarah-clements I tried doing this also But I was getting errors. 
Let me try this again and if errors persist then I will share the error here.

Reply:
If you get errors then try googling the error message or for different ways to achieve he line break :)

Reply:
The error is gone. It was not related to JSX or rendering thing. It was a chartjs thing and chartJS [doesn't allow you to render any HTML element](https://stackoverflow.com/a/42095510/5864458) from a callback function. That is why nothing was working. 
Finally Fixed! :) 
Thanks.